src/IntegratorConfig.php

```diff
@@ -29,7 +29,7 @@ class IntegratorConfig
     /**
      * @var string
      */
-    public const MANIFEST_KEY_DEFAULT_VALUE = 'default';
+    public const MANIFEST_KEY_DEFAULT_VALUE = 'defaultValue';
```

Comment:
Why did this change?

Reply:
because in https://github.com/spryker-sdk/integrator/blob/master/docs/manifest-types.md uses `defaultValue` key for `default` value or there are must be `default` key?

Reply:
Hi @Sento2012, could you please recheck manifest generation logic as I don't see the logic related to the `defaultValue` and `choices` options?

Reply:
Also, please extend tests with the above options.

Reply:
Added tests for `defaultValue` and `choices`